src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/Address.java

```diff
@@ -153,16 +153,28 @@ public void setValidUntil(LocalDate validUntil) {
 
     @JsonIgnore
     public AddressType getAddressType() {
-        if (Objects.nonNull(this.getValidUntil()) && isPastDate(this.getValidUntil())) {
-            return AddressType.PREVIOUS;
+
+        boolean validFromNotSpecified = (Objects.isNull(this.getValidFrom()));
+        boolean validFromIsSpecified = !validFromNotSpecified;
+        boolean validUntilNotSpecified = (Objects.isNull(this.getValidUntil()));
+        boolean validUntilIsSpecified = !validUntilNotSpecified;
+
+        // Time range is inverted
+        if ((validFromIsSpecified && validUntilIsSpecified)
```

Comment:
This conditional statement can be removed 
& the bottom conditional statement can be strengthened with a slight tweak (see below )